.github/workflows/build.yml

```diff
@@ -141,9 +141,9 @@ jobs:
     uses: howsoai/.github/.github/workflows/pytest.yml@main
     secrets: inherit
     with:
-      platform: 'macos-latest-xlarge'
+      platform: 'macos-latest-large'
       platform-pretty: 'MacOS'
-      amalgam-plat-arch: 'darwin-arm64'
+      amalgam-plat-arch: 'darwin-amd64'
       python-version: '3.11'
       config-fp: './config/latest-mt-debug-howso.yml'
```

Comment:
You also need a different config when changing from arm64 to amd64, one with 'no-avx'. I believe you did this already in another repo, can't remember which...

Reply:
Shoot, always forget about that. Thanks